pkg/agent/monitortool/monitor.go

```diff
@@ -47,8 +47,7 @@ const (
 	ipv6ProtocolICMPRaw = "ip6:ipv6-icmp"
 	protocolICMP        = 1
 	protocolICMPv6      = 58
-)
-
+      )
```

Comment:
this doesn't seem right
at least this doesn't match the style / format conventions we used in this code base